cars/cars.js

```diff
@@ -0,0 +1,86 @@
+const express = require("express");
+const router = express.Router();
+
+const db = require("./carsDb");
+router.use(express.json());
```

Comment:
`router.use(express.json());` is not needed here anymore, since you already applied json middleware in server.js (line 5)